java/Makefile

```
@@ -2124,6 +2127,21 @@ else
 endif
 
 
+###########################################################################
+### Diffs
+###
+
+# Don't diff StackMapUtils24.java, ClassGen24.java, MethodGen24.java, or
+# OperandStack24.java, because there are too many differences between the
+# upstream version (in BCEL or in bcel-util) and the version in Daikon.
+.PHONY: diff24
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)